src/wily/commands/graph.py

```diff
@@ -58,7 +57,7 @@ def graph(
     y_metric = resolve_metric(metrics[0])
     title = f"{x_axis.capitalize()} of {y_metric.description} for {path}{' aggregated' if aggregate else ''}"
 
-    if abs_path.is_dir() and not aggregate:
+    if ":" not in path and not aggregate:
```

Comment:
`colon:colon` is a valid (yet rare) directory name on macOS.

Reply:
That would probably break other principles, e.g. granularity is currently supported through the use of colons.

Reply:
cc45c525cb7437dca99707f61625d282c50e1b3d